apps/builder/app/builder/features/breakpoints/find-initial-width.test.ts

```diff
@@ -13,11 +13,31 @@ const breakpoints = [
 ];
 
 describe("Find initial width", () => {
-  test("base", () => {
+  test("base default", () => {
```

Comment:
Tests should contain no min|max breakpoints,
no breakpoints except Base at all.
Otherwise most of your logic is not covered with test at all

Reply:
I think we should always have the base breakpoint